alpine-rails-nginx/etc/nginx/nginx.conf

```diff
@@ -76,6 +66,7 @@ http {
 
   more_set_headers "X-Content-Type-Options: nosniff";
   more_set_headers "X-Frame-Options: deny";
+  more_set_headers "Content-Security-Policy: frame-ancestors 'none'"
```

Comment:
@dkubb Seems the presence of the `Content-Security-Policy` activates a whitelist vs a blacklist.

So we are "fine" by default. 

I'll research for the most strict default value, and if we have to explicitly blacklist `frame-ancestors`.